pkg/strategy/xgap/strategy.go

```diff
@@ -306,10 +306,13 @@ func buildAdjustPositionOrder(
 		pv = bestBid
 		side = types.SideTypeSell
 	}
+	price := pv.Price
+	quantity := fixedpoint.Min(
+		positionSnapshot.Base.Abs(),
```

Comment:
What if position is less than min quantity?

Reply:
Good point.

I think I should only making the quantity be no larger than the position in this function.
Then leave the check on dust quantity to the caller, which will be the check at L343.

Reply:
fixed.